app/models/miner.py

```diff
@@ -1,36 +1,30 @@
 import time
 from typing import Union
 
-from sqlalchemy import Column, String, Integer, BigInteger
+from sqlalchemy import Column, String, BigInteger
 
 from app import wrapper
 
 
 class Miner(wrapper.Base):
-    __tablename__: str = 'miner'
+    __tablename__: str = "miner"
 
     uuid: Union[Column, str] = Column(String(36), primary_key=True, unique=True)
     wallet: Union[Column, str] = Column(String(36))
     started: Union[Column, int] = Column(BigInteger)
-    power: Union[Column, int] = Column(Integer)
```

Comment:
keep this column and add a new `mcs` (morphcoins per second) column. the `power` column is neeeded for the `miner/get` endpoint and the `mcs` column is needed to store the current miner rate which also depends on the available hardware resources